cypress/fixtures/stubbing/default/workspaces.js

```diff
@@ -19,8 +29,95 @@ export const WORKSPACE_EXAMPLE = {
   sendInputToDataWarehouse: true,
   useDedicatedEventHubNamespace: true,
   sendScenarioMetadataToEventHub: true,
-  security: { default: 'admin', accessControlList: [] },
+  security: { default: ROLES.SCENARIO.ADMIN, accessControlList: usersAccess },
 };
 
 export const DEFAULT_WORKSPACE = WORKSPACE_EXAMPLE;
-export const DEFAULT_WORKSPACES_LIST = [DEFAULT_WORKSPACE];
+
+export const DEFAULT_WORKSPACES_LIST = [
```

Comment:
Do we really need 12 workspaces ?
Also, the `DEFAULT_WORKSPACES_LIST` should be the most simple case to run most of our tests, it should probably be a list with only one workspace in it.

Reply:
The purpose of the `DEFAULT_WORKSPACES_LIST` is also to test the graphic features of the workspace selector. So, it should be convenient to test an "extreme" case.

Reply:
So, I will create an EXTENTED_WOKSPACES_LIST